src/pages/_assets/welcome-content.tsx

```diff
@@ -0,0 +1,263 @@
+import { CardItem } from "../../components/WelcomeCard/welcome-card"
+
+// Section 1: Identity Management
+export const identityManagement: {
+  id: string
+  title: string
+  description: string
+  cards: CardItem[]
+} = {
+  id: "identity-management-section",
+  title: "Identity Management",
+  description: "What your users need for authentication and user management",
+  cards: [
+    {
+      title: "Implement User Login",
+      description: "Add email/password authentication to your application",
+      language: "auth",
+      languageLogoAlt: "Authentication icon",
+      docs: "/docs/kratos/concepts/credentials/username-email-password",
```

Comment:
I would sort of expect a guide here how to integrate Ory, but the document explains how to enable password login.